JSTests/test262/expectations.yaml

```diff
@@ -5,6 +5,9 @@ test/annexB/built-ins/Function/createdynfn-html-close-comment-params.js:
 test/annexB/built-ins/Function/createdynfn-html-open-comment-params.js:
   default: "SyntaxError: Unexpected token '}'. Expected a parameter pattern or a ')' in parameter list."
   strict mode: "SyntaxError: Unexpected token '}'. Expected a parameter pattern or a ')' in parameter list."
+test/annexB/built-ins/RegExp/prototype/compile/duplicate-named-capturing-groups-syntax.js:
```

Comment:
I think there's a `regexp-duplicate-named-groups` feature for this.

Reply:
Added.